server/liveblog/client_modules/client_modules.py

```diff
@@ -65,7 +70,16 @@ class ClientBlogsResource(BlogsResource):
 
 
 class ClientBlogsService(BaseService):
-    pass
+    def find_one(self, req, **lookup):
+        if req is None:
+            req = ParsedRequest()
+        cache_key = 'lb_ClientBlogService_get_%s' % (hash(frozenset(req.__dict__.items())))
```

Comment:
And I think that the generation of the key should be in the liveblog_cache class